test/test_other.py

```diff
@@ -4779,6 +4779,13 @@ def test(cxx, no_exit, assertions, flush=0, keepalive=0, filesystem=1):
           for flush in [0, 1]:
             test(cxx, no_exit, assertions, flush)
 
+  def test_opt_levels(self):
+    for opt in ['-O1', '-O2', '-Os', '-Oz', '-O3', '-Og', '-Ofast']:
```

Comment:
```suggestion
    for opt in ['-Og', '-Ofast']:
```

Reply:
Hmm, perhaps this test could be renamed something like `test_rare_opt_levels` to just give coverage of the rare ones? (that don't have lots of other coverage) I think then it would be self-consistent.